packages/coding-standards/Jetpack/ruleset.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Jetpack" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/squizlabs/PHP_CodeSniffer/master/phpcs.xsd">
+
+	<rule ref="PHPCompatibilityWP"/>
+	<rule ref="WordPress-Core" />
```

Comment:
WordPress-Core not needed as WordPress-Extra already includes it.

Reply:
Fixed in e64898095